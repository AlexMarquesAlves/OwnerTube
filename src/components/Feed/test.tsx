import { render, screen } from "@testing-library/react";
import { expect, it } from "vitest";
import { Feed } from ".";

it("should render Feed with text Hello", () => {
  render(<Feed>Hello</Feed>);

  const heading = screen.getByRole("heading", { name: /Hello/ });

  expect(heading).toBeInTheDocument();
  expect(heading).toMatchSnapshot();
});
