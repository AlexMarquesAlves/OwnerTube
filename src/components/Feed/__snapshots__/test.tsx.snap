// Vitest Snapshot v1

exports[`should render Feed with text Hello 1`] = `
<h1>
  Feed
  Hello
</h1>
`;
